client/blocks/app-banner/index.jsx

```diff
@@ -228,13 +233,15 @@ export function buildDeepLinkFragment( currentRoute, currentSection ) {
 const mapStateToProps = state => {
 	const sectionName = getSectionName( state );
 	const isNotesOpen = isNotificationsOpen( state );
+	const isInActivityLog = window.location.pathname.indexOf( '/stats/activity/' ) !== -1;
```

Comment:
do we not have this info in state as something like `getSection`? or `getPath`?

also we should be able to use `String.includes()` now since we started polyfilling it. you'd want to test to make sure

```js
window.location.pathname.includes( '/stats/activity' )
```

Reply:
See https://github.com/Automattic/wp-calypso/pull/26286#discussion_r204923202